.github/actions/e2e_sonobuoy/action.yml

```diff
@@ -48,6 +48,12 @@ runs:
         sonobuoy results *_sonobuoy_*.tar.gz
         sonobuoy results *_sonobuoy_*.tar.gz --mode detailed | jq 'select(.status!="passed")' | jq 'select(.status!="skipped")' || true
 
+    - name: Cleanup sonobuoy deployment
+      env:
+        KUBECONFIG: ${{ inputs.kubeconfig }}
+      shell: bash
+      run: sonobuoy delete --wait
+
```

Comment:
Is there a particular reason to add this? I fear that this is just one more source of sporadics, which we could avoid because the cluster is shut down afterwards anyway.

Reply:
We can drop it. I just kept the manual from your last upstream PR